Dockerfile.lint

```diff
@@ -0,0 +1,15 @@
+FROM registry.redhat.io/ubi8/ubi-minimal:8.7-1031 as builder
+
+RUN NODEJS_VERSION=16 && \
+    microdnf module disable nodejs && \
+    microdnf module enable nodejs:$NODEJS_VERSION && \
+    microdnf --nodocs install npm && \
+    microdnf clean all && \
+    rm -rf /mnt/rootfs/var/cache/* /mnt/rootfs/var/log/dnf* /mnt/rootfs/var/log/yum.*
+
+WORKDIR /usr/src/app
+
+COPY . .
+RUN npm install eslint
```

Comment:
not sure if only eslint is needed as we night need some eslint plugins. I think it will be easier to build an image for tests, then we can run lint or unit tests from it